app/views/shared/_authentication_actions.html.erb

```diff
@@ -1,4 +1,4 @@
-<a href="<%= sign_up_path(state: "new-user") %>" class="crayons-btn" aria-label="Create new account">
+<a href="<%= sign_up_path(state: "new-user") %>" class="crayons-btn" aria-label="Create new account" data-no-instant>
```

Comment:
This is the point of contention, described in a video: https://imgur.com/a/diTOh8g